src/framework/dockwindow/thirdparty/KDDockWidgets/src/private/DragController.cpp

```diff
@@ -738,6 +788,16 @@ bool DragController::eventFilter(QObject *o, QEvent *e)
         }
     }
 
+    switch (e->type()) {
+    case QEvent::KeyPress:
+    case QEvent::KeyRelease: {
+        QKeyEvent *ke = static_cast<QKeyEvent *>(e);
+        return activeState()->handleKeyPressRelease(ke);
```

Comment:
Similar to how we handle the Wayland stuff, I think we should only return if the key is intercepted/handled:

```suggestion
        if (activeState()->handleKeyPressRelease(static_cast<QKeyEvent *>(e)) {
            return true;
        }
```